### PR TITLE
[T4A2][T13-B3] Annabel Eng Jing Wen

### PR DESCRIPTION
**Pull request by annabeleng**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -81,7 +81,8 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
+            new Parser();
```
Comment (z11i) [inline src/seedu/addressbook/Main.java:84]: If all of your methods in the Parser class are already `static`, do we still need to initialize an instance of Parser? Think about how we normally use the Math class in Java API.

File: test/java/seedu/addressbook/parser/ParserTest.java [hunk 2]
```diff
@@ -18,11 +18,9 @@
 
 public class ParserTest {
 
-    private Parser parser;
-
     @Before
     public void setup() {
-        parser = new Parser();
+        new Parser();
```
Comment (zhongwei-z) [inline test/java/seedu/addressbook/parser/ParserTest.java]: No need to initialize Parser.

**Comment by z11i** [comment]
In class you seemed to be confused about class level variables/methods. Do check out our handouts and other resources for clarification. Let me know in the next tutorial if you still have doubts.

**Comment by annabeleng** [comment]
Hi @zhongwei-z ZhongWei, I am more clear about this concept after asking around and googling. Thanks for your explanation in class too. (:

**Pull request closed without merging**